CorsixTH/Lua/game_ui.lua

```diff
@@ -1112,10 +1098,6 @@ function GameUI:afterLoad(old, new)
   if old < 23 then
     self.do_world_hit_test = not self:getWindow(UIPlaceObjects)
   end
-  if old < 28 then
```

Comment:
Do I actually need to keep this?

Reply:
You probably want to drop it since you handle this in your < 128 condition.